addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2209,12 +2209,16 @@ export class OdooEditor extends EventTarget {
         }
     }
     _positionToolbar() {
+        const sel = this.document.getSelection();
+        if (!sel || !sel.rangeCount) {
+            console.log('Selection is null, toolbar should not be replaced');
```

Comment:
👀

Reply:
whoops, my bad! I was using this as a temp solution cause the edge case was triggering another two problems from the color picker. I created another task about the color picker, was fixing that one first and marked this one as green :grimacing:

I tested it again and the issue is actually is when applying color, the selection isn't recovered properly. I will make an actual fix and move this task in RD Dev :pray: